### PR TITLE
feat: [CO-767] Remove stale config from Carbonio SOAP Service config 

### PR DESCRIPTION
**Pull request by keshavbhatt**
- all zimlet related config was removed
- change the display name of service to Carbonio SOAP Service

### CODE REVIEW

File: store/conf/web.xml [hunk 1]
```diff
@@ -82,17 +82,6 @@ SPDX-License-Identifier: AGPL-3.0-only
     <url-pattern>/*</url-pattern>
   </filter-mapping>
 
-  <filter>
-    <filter-name>ThrottlingFilter</filter-name>
-    <filter-class>com.zimbra.cs.servlet.ThrottlingFilter</filter-class>
```
Comment (frisonisland) [inline store/conf/web.xml:87]: What about removing the class completely?

**Review by frisonisland — COMMENTED** [review]
Just want to know if we can remove also zimlet Filter classes

**Comment by keshavbhatt** [comment]
> Just want to know if we can remove also zimlet Filter classes

We will cover that in CO-778

File: store/conf/web.xml (hunk 1 above)
Comment (keshavbhatt) [reply store/conf/web.xml:87]: We will cover that in CO-778

**Review by frisonisland — APPROVED** [review]

**Review by Polpetta — APPROVED** [review]

**Pull request merged**